### ISSUE TITLE
Automatic Tribe Sharing?

### ISSUE THREAD

**Issue by MrBazibaz** [cadon/ARKStatsExtractor#1257]
Hello.

I have been trying to get our tribe files to automatically share between us.
I've setup a shared folder where everyone has their savepath.
currently we have to press file -> load and add... and then their save file.
Is it possible to make the program do this process automatically? so it syncs between the saves?

**Comment by coldino** [comment]
Yup, it's pretty easy. Enable the settings for auto-load and auto-save, then all users can simply load the same synced file and work as usual.

Note that some sync services like Google Drive are extremely slow and will cause many issues while others like Dropbox are nearly instant and are used successfully by many ASB users.
